README.md

```diff
@@ -90,6 +90,18 @@ curl -Lo ec2-metadata-mock https://github.com/aws/amazon-ec2-metadata-mock/relea
 curl -Lo ec2-metadata-mock https://github.com/aws/amazon-ec2-metadata-mock/releases/download/v0.9.3/ec2-metadata-mock-windows-amd64
 ```
 
+### Docker
+```
+docker pull amazon/amazon-ec2-metadata-mock:v0.9.3
+```
+
+Note, when running on Windows or Mac, you will need to change the default bind hostname from localhost because the binary runs in
+a virtual machine, e.g.,
+
+```
+docker run -it --rm amazon/amazon-ec2-metadata-mock:v0.9.3 --hostname 0.0.0.0
```

Comment:
* passing hostname "" could add flexibility, allowing `localhost` too
* I had to bind host port to container port to make the above command work - `docker run -it -p 1338:1338 --rm amazon/amazon-ec2-metadata-mock:v0.9.3 --hostname ""`

Reply:
@pdk27 , what are your thoughts on updating the [default value](https://github.com/aws/amazon-ec2-metadata-mock/blob/master/pkg/config/server.go#L23) from 'localhost' to '0.0.0.0'  OR "" (empty string)? Then we wouldn't need this additional arg.

From the [documentation](https://golang.org/pkg/net/#Listen), localhost should still be valid if hostname is empty:

> if the host in the address parameter is empty or a literal unspecified IP address, Listen listens on all available unicast and anycast IP addresses of the local system.

also tested locally and confirmed localhost is still valid.

Reply:
I think a default of `0.0.0.0` (or whatever allows it to bind to any address) makes sense since this is for testing and an overly restrictive bind doesn't help much. One reference point is the mysql docker, which defaults to binding to any address https://hub.docker.com/_/mysql

Reply:
I think we are leaning towards that as well. Going to finish discussion offline so others can benefit from the updated Docker installation instructions while we make the changes.